e2e/src/scenarios/print.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+import { AppPage } from '../app.po';
+import { browser, protractor } from 'protractor';
+
+describe('print', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+
+  it('should print the pdf', () => {
+    page.getPdfViewer().click();
+
+    const printPage = page.getPrintButton();
+    printPage.click();
+
+    // clean up by moving to app.po
+    browser.getAllWindowHandles().then(function(handles) {
+      expect(handles.length).toEqual(2);
+      browser.switchTo().window(handles[1]);
+      expect(page.getPrintDialog).toBeTruthy();
+      page.wait(1000);
+      browser.close();
```

Comment:
weird how you close the browser, then switch to another tab in the browser. Is there any way to do something like `browser.close(window(handles[1]))`?

Reply:
nvm, I couldn't find anything better than what you've got